src/ddl/migrations/covid_hosp_state_split_tables.sql

```diff
@@ -0,0 +1,148 @@
+  `previous_day_admission_adult_covid_suspected_30_39` INT,
+  `previous_day_admission_adult_covid_suspected_30_39_coverage` INT,
+  `previous_day_admission_adult_covid_suspected_40_49` INT,
+  `previous_day_admission_adult_covid_suspected_40_49_coverage` INT,
+  `previous_day_admission_adult_covid_suspected_50_59` INT,
+  `previous_day_admission_adult_covid_suspected_50_59_coverage` INT,
+  `previous_day_admission_adult_covid_suspected_60_69` INT,
+  `previous_day_admission_adult_covid_suspected_60_69_coverage` INT,
+  `previous_day_admission_adult_covid_suspected_70_79` INT,
+  `previous_day_admission_adult_covid_suspected_70_79_coverage` INT,
+  `previous_day_admission_adult_covid_suspected_80plus` INT,
+  `previous_day_admission_adult_covid_suspected_80plus_coverage` INT,
+  `previous_day_admission_adult_covid_suspected_unknown` INT,
+  `previous_day_admission_adult_covid_suspected_unknown_coverage` INT,
+  `deaths_covid` INT,
+  `deaths_covid_coverage` INT,
+  `on_hand_supply_therapeutic_a_casirivimab_imdevimab_courses` INT,
+  `on_hand_supply_therapeutic_b_bamlanivimab_courses` INT,
+  `on_hand_supply_therapeutic_c_bamlanivimab_etesevimab_courses` INT,
+  `previous_week_therapeutic_a_casirivimab_imdevimab_courses_used` INT,
+  `previous_week_therapeutic_b_bamlanivimab_courses_used` INT,
+  `previous_week_therapeutic_c_bamlanivimab_etesevimab_courses_used` INT,
+  `icu_patients_confirmed_influenza` INT,
+  `icu_patients_confirmed_influenza_coverage` INT,
+  `previous_day_admission_influenza_confirmed` INT,
+  `previous_day_admission_influenza_confirmed_coverage` INT,
+  `previous_day_deaths_covid_and_influenza` INT,
+  `previous_day_deaths_covid_and_influenza_coverage` INT,
+  `previous_day_deaths_influenza` INT,
+  `previous_day_deaths_influenza_coverage` INT,
+  `total_patients_hospitalized_confirmed_influenza` INT,
+  `total_patients_hospitalized_confirmed_influenza_covid` INT,
+  `total_patients_hospitalized_confirmed_influenza_covid_coverage` INT,
+  `total_patients_hospitalized_confirmed_influenza_coverage` INT,
+  PRIMARY KEY (`id`),
+  -- for uniqueness
+  -- for fast lookup of most recent issue for a given state and date
+  UNIQUE KEY `issue_by_state_and_date` (`state`, `date`, `issue`),
+  -- for fast lookup of a time-series for a given state and issue
+  KEY `date_by_issue_and_state` (`issue`, `state`, `date`),
+  -- for fast lookup of all states for a given date and issue
+  KEY `state_by_issue_and_date` (`issue`, `date`, `state`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- 2. Move data with record_type=D into new table
+
+INSERT INTO `covid_hosp_state_daily`
+SELECT * FROM `covid_hosp_state_timeseries`
+WHERE record_type='D';
+
+-- 3. Remove that data from the old table
+
+DELETE FROM `covid_hosp_state_timeseries` WHERE record_type='D';
+
+-- 4. Remove the record_type column from both tables
+
+ALTER TABLE `covid_hosp_state_daily` DROP COLUMN record_type;
+ALTER TABLE `covid_hosp_state_timeseries` DROP COLUMN record_type;
```

Comment:
record_type is included in some indexes in `covid_hosp_state_timeseries`...  does this DROP COLUMN make any complaints related to that?

Reply:
Nope, did a quick test and this ran with no warnings:

```
mysql> ALTER TABLE `covid_hosp_state_daily` DROP COLUMN record_type;
Query OK, 0 rows affected (0.03 sec)
Records: 0  Duplicates: 0  Warnings: 0

mysql> ALTER TABLE `covid_hosp_state_timeseries` DROP COLUMN record_type;
Query OK, 0 rows affected (0.03 sec)
Records: 0  Duplicates: 0  Warnings: 0
```